readme.md

```diff
@@ -2,12 +2,10 @@
 
```

Comment:
- [ ] Great work but I don't see any change in the code files related to the redux toolkit so i cannot proceed a review on your code 🤔